front50-api/front50-api.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'java-library'
+
+dependencies {
+  implementation project(":front50-core")
```

Comment:
Unfortunately this drags in all of `front50-core`'s dependencies. I think you need this just because of the `Pipeline` model. `Pipeline` should come in to this package (and it will end up dragging in `Timestamped` too).

Those classes all have Jackson annotations. To get around pulling in Jackson as a dependency, use mixins: https://github.com/FasterXML/jackson-docs/wiki/JacksonMixInAnnotations